e2e/tests-dfx/install.bash

```diff
@@ -81,3 +81,53 @@ teardown() {
     dfx canister create --all
     assert_command_fail dfx canister install --all --wasm "${archive:?}/wallet/0.10.0/wallet.wasm"
 }
+
+@test "install runs post-install tasks" {
+    dfx_start
+    # shellcheck disable=SC2094
+    cat <<<"$(jq '.canisters.e2e_project.post_install="sh -c \"echo hello\""' dfx.json)" >dfx.json
```

Comment:
Should these have the same kind of handling that the custom build script field does?  All of these post-install scripts are running `sh -c "something"`, but if the post-install script was "post-install.sh", would you expect that to work without a leading `./`?